configuration/concepts/diagnoses-set-and-answers.csv

```diff
@@ -0,0 +1,8 @@
+uuid,Void/Retire,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Data class,Data type,Mappings|SAME-AS|PIH|Name,Mappings|SAME-AS|PIH|Number,Mappings|SAME-AS|CIEL,Mappings|SAME-AS|AMPATH,Mappings|NARROWER-THAN|AMPATH,Mappings|SAME-AS|ICD-10-WHO,Mappings|NARROWER-THAN|ICD-10-WHO,Mappings|BROADER-THAN|ICD-10-WHO,Mappings|NARROWER-THAN|ICD-10-WHO 2nd,Mappings|SAME-AS|ICD-11-WHO,Mappings|SAME-AS|Liberia MoH,Mappings|NARROWER-THAN|Liberia MoH,Mappings|SAME-AS|org.openmrs.module.emrapi,Mappings|SAME-AS|PIH Malawi,Mappings|SAME-AS|SNOMED CT,Mappings|NARROWER-THAN|SNOMED CT
```

Comment:
This set contains the master "set of sets" concepts and 6 "answer" concepts that are used by a few of the diagnoses.

It's possible we don't need to explicitly add "Yes", "No", and "Unknown" here, but just assume they are installed by the base package.

As for the other three random concepts, "SINUS VENOUS", "SECUNDUM" & "PRINUM" there's a single diagnosis in ("Atrial septal defect") the primary care set that has the as answers.

Its very likely we aren't using this concept as a question in Mexico, so we could potentially just remove these and the concept answers on the "Atrial septal defect" concept, though of course this would make the concept deviate in Mexico from what is on our concepts server.